### PR TITLE
Update podcast-feed-opus.rss

### PR DESCRIPTION
**Pull request by cartierbresson**
@ltsavar & @jospoortvliet 🥇

### CODE REVIEW

File: podcast-feed-opus.rss [hunk 1]
```diff
@@ -38,6 +38,22 @@
...
     <atom:link href="https://nextcloud.com/podcast-feed.xml" rel="self" type="application/rss+xml" />
     <link>https://nextcloud.com/podcast/</link>
+      
+          <item>
+      <title>Podcast No. 8 - Interview with Arthur Schiwon about Nextcloud Flow</title>
+      <itunes:subtitle> In this eight-episode, our host Ingo talked with Arthur, the Senior Software Engineer at Nextcloud about Nextcloud Flow.</itunes:subtitle>
+      <itunes:summary>In this eight-episode, our host Ingo talked with Arthur, the Senior Software Engineer at Nextcloud about Nextcloud Flow. Nextcloud Flow is the workflow engine of Nextcloud. 
+      </itunes:summary>
+      <itunes:duration>00:25:39/itunes:duration>
+      <description>In this eight-episode, our host Ingo talked with Arthur, the Senior Software Engineer at Nextcloud about Nextcloud Flow. Nextcloud Flow is the workflow engine of Nextcloud. </description>
```
Comment (ltsavar) [inline podcast-feed-opus.rss:48]: </itunes:duration>

File: podcast-feed-opus.rss [hunk 2]
```diff
@@ -38,6 +38,22 @@
...
+      <pubDate>Tue, 26 Feb 2021 9:00:00 GMT</pubDate>
```
Comment (ltsavar) [inline podcast-feed-opus.rss:49]: Thu

**Review by ltsavar — CHANGES_REQUESTED** [review]
2 little things...

File: podcast-feed-opus.rss [hunk 3]
```diff
@@ -38,6 +38,22 @@
     <language>en-us</language>
     <atom:link href="https://nextcloud.com/podcast-feed.xml" rel="self" type="application/rss+xml" />
     <link>https://nextcloud.com/podcast/</link>
+      
+          <item>
+      <title>Podcast No. 8 - Interview with Arthur Schiwon about Nextcloud Flow</title>
+      <itunes:subtitle> In this eight-episode, our host Ingo talked with Arthur, the Senior Software Engineer at Nextcloud about Nextcloud Flow.</itunes:subtitle>
+      <itunes:summary>In this eight-episode, our host Ingo talked with Arthur, the Senior Software Engineer at Nextcloud about Nextcloud Flow. Nextcloud Flow is the workflow engine of Nextcloud. 
+      </itunes:summary>
+      <itunes:duration>00:25:39/itunes:duration>
```
Comment (jospoortvliet) [inline podcast-feed-opus.rss:47]:
```suggestion
      <itunes:duration>00:25:39</itunes:duration>
```

File: podcast-feed-opus.rss (hunk 1 above)
Comment (jospoortvliet) [inline podcast-feed-opus.rss:48]:
```suggestion
      <description>In this eight-episode, our host Ingo talked with Arthur, Senior Software Engineer at Nextcloud, about Nextcloud Flow. Nextcloud Flow is the workflow engine of Nextcloud. </description>
```

File: podcast-feed-opus.rss [hunk 4]
```diff
@@ -38,6 +38,22 @@
     <language>en-us</language>
     <atom:link href="https://nextcloud.com/podcast-feed.xml" rel="self" type="application/rss+xml" />
     <link>https://nextcloud.com/podcast/</link>
+      
+          <item>
+      <title>Podcast No. 8 - Interview with Arthur Schiwon about Nextcloud Flow</title>
+      <itunes:subtitle> In this eight-episode, our host Ingo talked with Arthur, the Senior Software Engineer at Nextcloud about Nextcloud Flow.</itunes:subtitle>
+      <itunes:summary>In this eight-episode, our host Ingo talked with Arthur, the Senior Software Engineer at Nextcloud about Nextcloud Flow. Nextcloud Flow is the workflow engine of Nextcloud.
```
Comment (jospoortvliet) [inline podcast-feed-opus.rss:45]:
```suggestion
      <itunes:summary>In this eight-episode, our host Ingo talked with Arthur, Senior Software Engineer at Nextcloud. about Nextcloud Flow. Nextcloud Flow is the workflow engine of Nextcloud. 
```

File: podcast-feed-opus.rss (hunk 2 above)
Comment (jospoortvliet) [inline podcast-feed-opus.rss:49]:
```suggestion
      <pubDate>Fri, 26 Feb 2021 9:00:00 GMT</pubDate>
```

**Review by jospoortvliet — CHANGES_REQUESTED** [review]

**Review by jospoortvliet — APPROVED** [review]

**Pull request merged**